agent/engine/docker_container_engine.go

```diff
@@ -39,7 +39,8 @@ import (
 )
 
 const (
-	dockerDefaultTag = "latest"
+	dockerDefaultTag          = "latest"
+	containerEventsBufferSize = 1000
```

Comment:
Why was this changed? Please add a description in either the commit message or as inline documentation. Doesn't this open up the possibility of the ECS Agent processing multiple events from docker simultaneously? Have you analyzed the impact of the same on the container and task engines?

Reply:
The description is added in the Summary, I will add the same in the commit message. 
I don't think ECS Agent can process multiple events simultaneously, based on the [code](https://github.com/aws/amazon-ecs-agent/blob/master/agent/engine/docker_task_engine.go#L403) the events will be handled serially. But I could be wrong, please correct me if that's not true.
>Have you analyzed the impact of the same on the container and task engines?

Can you elaborate here, I don't think this will have impact on the container or task engine since the behavior of handling this doesn't change. The event will still be handled serially in task engine.